packages/components/src/Tabs/Tabs.css

```diff
@@ -11,7 +11,8 @@
 
 .overflow {
   position: relative;
-  border-bottom: solid var(--border-base) var(--color-border);
+  padding-left: var(--public-tab--inset);
```

Comment:
removed the extra border because it is also on `.tabRow`. I did this so that the borderBottom wouldn't extend too far out.